x-pack/plugins/infra/public/components/asset_details/hooks/use_page_header.tsx

```diff
@@ -148,7 +171,7 @@ const useTabs = (tabs: Tab[]) => {
 
   const tabEntries: TabItem[] = useMemo(
     () =>
-      tabs.map(({ name, ...tab }) => {
+      tabs.filter(isTabEnabled).map(({ name, ...tab }) => {
```

Comment:
Nice ✨

Reply:
yeah! thanks for pointing that out :). I like this version much better too.